app/src/main/java/de/tum/in/tumcampusapp/models/transport/TransportFavorites.kt

```diff
@@ -0,0 +1,9 @@
+package de.tum.`in`.tumcampusapp.models.transport
+
+import android.arch.persistence.room.Entity
+import android.arch.persistence.room.PrimaryKey
+
+@Entity(tableName = "transport_favorites")
+data class TransportFavorites(@PrimaryKey(autoGenerate = true)
+                              var id: Int = 0,
+                              var symbol: String = "")
```

Comment:
shouldn't the symbol be unique? can't we even set it as primarykey?

Reply:
Autogenerate was set to true for the PrimaryKey "id". If we want to set "symbol" as PrimaryKey as well, we should use @Entity(primaryKeys = ["id","symbol"] (proper way to declare composite PrimaryKeys in Room otherwise there will be an error: You cannot have multiple primary keys defined in an Entity...). But then the unique ids for the column "id" are not generated automatically anymore.

Reply:
I thought more of a UNIQUE column...
https://stackoverflow.com/a/46035385

Composite primary keys wouldn't help much, you're right...